.github/actions/setup-chart-testing-environment/action.yml

```diff
@@ -19,47 +19,39 @@ runs:
   steps:
     - name: Set up Helm
       id: setup-helm
-      uses: azure/setup-helm@18bc76811624f360dbd7f18c2d4ecb32c7b87bab # v1.1
+      uses: azure/setup-helm@b5b231a831f96336bbfeccc1329990f0005c5bb1 # v3.3
       with:
-        version: v3.8.1
+        version: "latest"
 
-    - uses: actions/setup-python@v2
+    - uses: actions/setup-python@v4
       id: setup-python
       with:
-        python-version: 3.7
+        python-version: 3.8
 
     - name: Set up chart-testing
       id: setup-chart-testing
-      uses: helm/chart-testing-action@5f16c27cf7a4fa9c776ff73734df3909b2b65127 # v2.1.0
+      uses: helm/chart-testing-action@09ed88797198755e5031f25be13da255e7e33aad  # v2.3.0
       with:
         version: v3.5.0
 
     - name: Create minikube Kubernetes ${{ inputs.k8s_version }} Cluster
       id: create-minikube-cluster
-      # TODO: Workaround until conditions are supported natively
```

Comment:
GHA now natively supports step conditionals so this action is not needed anymore.